README.md

```diff
@@ -1,34 +1,18 @@
-# pytest-sugar
+# pytest-sugar ✨
 
-[![Build status](https://travis-ci.org/Teemu/pytest-sugar.svg?branch=master)](https://travis-ci.org/Teemu/pytest-sugar)
 [![Pypi version](https://img.shields.io/pypi/v/pytest-sugar.svg)](https://pypi.org/project/pytest-sugar/)
 
-pytest-sugar is a plugin for [pytest](http://pytest.org) that shows
-failures and errors instantly and shows a progress bar.
+This plugin extends [pytest](http://pytest.org) by showing failures and errors instantly, adding a progress bar, improving the test results and making the output look better.
 
-![Demo](https://i.imgur.com/jER0Jxj.gif)
-
-## Similar projects
-
-- [pytest-rich](https://github.com/nicoddemus/pytest-rich)
-- [pytest-pretty](https://github.com/samuelcolvin/pytest-pretty)
-
-## Requirements
-
-You will need the following prerequisites in order to use pytest-sugar:
-
-- Python 2.7, 3.4 or newer
-- pytest 2.9.0 or newer
-- pytest-xdist 1.14 or above if you want the progress bar to work while running
-  tests in parallel
+![render1667890332624-min](https://user-images.githubusercontent.com/53298/200600769-7b871b26-a36a-4ae6-ae24-945ee83fb74a.gif)
 
 ## Installation
 
 To install pytest-sugar:
 
     pip install pytest-sugar
 
-Then run your tests with:
+The plugin is activated by activated. Run your tests normally:
```

Comment:
`The plugin is activated by activated.` ➡️ `The plugin is activated automatically.` ?

I'll submit a PR to address this and a few other minor changes in just a moment.

Reply:
Oops